core/store/orm/config.go

```diff
@@ -210,6 +212,17 @@ func init() {
 		OptimismGasFees:                  true,
 	}
 
+	// Fantom
+	fantomTestnet := ChainSpecificDefaultSet{
```

Comment:
It is confusing, but the zero values are often not correct. So it might be necessary to either:

1. Set every field 
2. Copy from mainnet first and then overwrite certain values

Reply:
We probably need to come up with a less fragile way of doing this.